index.html

```diff
@@ -15,8 +15,8 @@ <h2 class="title-navbar">Developer Tools</h2>
   </nav>
   <div class="container">
     <h2 class="title">Front-End Tools</h2>
-    <div class="display-wrap">
-      <div class="tech-panel">
+    <div class="grid">
+      <div class="tech-panel border">
```

Comment:
Instead of applying a new `border` class, you can do it all in the `:first-child`:

```scss
  &:first-child {
    border-left: 1px solid $green;
    grid-row: 1/3;
  }
```